frontend/docsettings.lua

```diff
@@ -74,10 +74,18 @@ function DocSettings:readSetting(key)
     return self.data[key]
 end
 
+function DocSettings:get(key)
```

Comment:
Emmm, I suggest to change original function name instead of adding a new shortcut.